stable/index.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 entries:
```

Comment:
Putting this in stable will give access to everyone, do we need that? or should you add it to `repo/staging` first and let everyone use 
```
helm repo add https://raw.githubusercontent.com/portworx/helm/master/repo/staging
```
and helm install with --version as 1.3.0
?

Reply:
# helm repo list
NAME         	URL
portworx     	http://charts.portworx.io/
[root@ip-70-0-55-157 ~]# helm repo update
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "portworx" chart repository
Update Complete. ⎈Happy Helming!⎈

[root@ip-70-0-55-157 ~]# helm install px-backup portworx/px-backup --namespace px-backup1 --create-namespace --set persistentStorage.enabled=true,persistentStorage.storageClassName=test-sc --version=1.3.0
Error: failed to download "portworx/px-backup" (hint: running `helm repo update` may help)

The repo has been indexed with 1.3.0 like and also it will be merged into 1.3.0 branch.
helm repo index --url https://raw.githubusercontent.com/portworx/helm/1.3.0/stable/ .

So it should not be available under https://raw.githubusercontent.com/portworx/helm/master/stable/ before release.

Reply:
Just saw you are merging in 1.3.0 you can ignore this. rest LGTM